packages/trpc/server/routers/loggedInViewer/integrations.handler.ts

```diff
@@ -39,10 +36,6 @@ type TeamQuery = Prisma.TeamGetPayload<{
   };
 }>;
 
-// type TeamQueryWithParent = TeamQuery & {
```

Comment:
@joeauyeung removed this code which was commented but I don't if this is necessary or left there by mistake, should I bring it back?